sdks/java/io/common/src/test/java/org/apache/beam/sdk/io/common/IOTestPipelineOptions.java

```diff
@@ -88,4 +88,10 @@
   Integer getCassandraPort();
   void setCassandraPort(Integer port);
 
+  /* Text */
```

Comment:
<!--new_thread; commit:c6c7070ad92424707d3720d3a4dc2c0fb6961440; resolved:0-->
Change comment to "Options for test pipeline for TextIO in 'sdks/java/io/text/src/test/java/org/apache/beam/sdk/io/text/TextIOIT.java'."

Reply:
ok